### DISCUSSION TITLE
Android support

### DISCUSSION THREAD

**Discussion by friuns2** [hiperbou/kotlin-unreal#24] [General]
Hello did you tried to build on android? Seems latest unreal.js have some problems

https://github.com/ncsoft/Unreal.js/issues/323
https://github.com/ncsoft/Unreal.js-core/issues/43

**Comment by hiperbou** [comment]
I haven't tried to build any Unreal project on Android. Unreal.js maintainers do a great job at fixing bugs and updating to the latest Unreal versions, so I would expect them to fix any blocking problem like not being able to build. Of course, that probably would take some time.
